src/Components/leftside-bar/leftside-bar.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import "./leftside-bar.css";
+import RightsideBar from "../settings-dropdown/email-configuration"; // Adjust the import path as necessary
```

Comment:
**Clarity**
I'm confused here. It looks like we're importing RighsideBar from email-configuration but I couldn't find the RightsideBar in the email-configuration.ts file.

Am I missing something here?